docs/notebooks/Poincare Evaluation.ipynb

```diff
@@ -933,106 +933,6 @@
        "        <th>100</th>\n",
        "        <th>200</th>\n",
        "    </tr>\n",
-       "    <tr>\n",
```

Comment:
Why this removed?

Reply:
Again I wasn't able to complete the execution of this cell as it was taking too long

Reply:
This isn't good, this information **must be** in this notebook.

Reply:
Okay i will try running them again